### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by jamestucker1988**

### CODE REVIEW

**Comment by jamestucker1988** [comment]
THE given project is a curriculum based solo project where we have to clone apple archive page. I have divided my project into three section nav,main-section and footer.use of linear gradient is expected as such I have used it in my project.
**HERE is a link to live preview :<a href="https://raw.githack.com/jamestucker1988/apple-microverse-project/feature/index.html">click here</a>**

**Comment by sumancrest0001** [comment]
**Status: changes required
Nicely done @jamestucker1988.**
We are very close to getting a green signal for the project. But we need to make a couple of changes to make sure that it looks like the original one.
**[NOTE] All the changes are mandatory.**

**Comment by sumancrest0001** [comment]
We have to remove the horizontal scroll bar and the background image should cover the screen at any screen size.

- Your verse section is repeated.
- We have to also give a similar look to the footer section. Fix the font-size and color
Currently, it looks :point_down: 
Screenshot from 2020-05-13 11-04-49

It should be :point_down: 
Screenshot from 2020-05-13 11-05-15

File: style.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  border: none;
+  background-color: white;
+}
+
+.fa-search {
+  outline: none;
+  border: none;
+  background-color: white;
+  color: grey;
+  padding-top: 0.2rem;
```
Comment (sumancrest0001) [inline style.css:78]: We have to remove the repeated styling and follow the DRY principle. [Pathright requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) and [helpful article](https://speckyboy.com/good-bad-css-practices/)

**Comment by jamestucker1988** [comment]
THE given project is a curriculum based solo project where we have to clone apple archive page. I have divided my project into three section nav,main-section and footer.use of linear gradient is expected as such I have used it in my project.
The following changes have been made in the given draft:
1- the navbar is made  similar to the original one.
2- Gradient is from top to bottom . Also similar styling is given to nav items and search input field.
3-horizontal scroll is fixed.
4-tried to implement dry programming .
5-font size is changed as expected.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```
Comment (lucasmazo32) [inline index.html:1]: - [ ] Kindly fix the W3C validator errors.

> image

- [ ] In my screen it doens't display the whole page and it doesn't let me scroll either. Kindly check that.

> image

File: README.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+​
+![screenshot](images/webpage_preview_1.PNG)
+
+### Middle of the webpage
+
+![screenshot 2](images/webpage_preview_3.PNG)
+
+### Bottom of the webpage
+
+![screenshot 3](images/webpage_preview_4.PNG)
```
Comment (lucasmazo32) [inline README.md:19]: - [ ] Kindly check this section, it is not displaying this images.

> image

**Comment by jamestucker1988** [comment]
THE given project is a curriculum based solo project where we have to clone apple archive page. I have divided my project into three section nav,main-section and footer.use of linear gradient is expected as such I have used it in my project.
The following changes have been made in the given draft:
1-fixed validation errors
2-removed overflow-y.
3-included a preview of webpage.

**HERE is a link to live preview :<a href="https://raw.githack.com/jamestucker1988/apple-microverse-project/feature/index.htmll">click here</a>**

**Comment by sumancrest0001** [comment]
**STATUS: CHANGES REQUIRED
Hello @jamestucker1988**, You have done a good job to fix issues addressed by the previous TSE but we need to make a couple of minor changes to get approval for this project. Please make the following changes and all of them are mandatory.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        </nav>
+
+
+    </main>
+
+
+    <img src="image/promo_30_years.jpg" alt="">
+    <img src="image/promo_iphone5c.jpg" alt="">
+    <img src="image/promo_iphone5s.jpg" alt="">
+    <img src="image/promo_supplier_responsibility.jpg" alt="">
```
Comment (sumancrest0001) [inline index.html:38]: These articles should have a link. we have to use <a> tag. [link to original page](https://web.archive.org/web/20140301004610/http://www.apple.com/)

**Comment by jamestucker1988** [comment]
THE given project is a curriculum based solo project where we have to clone apple archive page. I have divided my project into three section nav,main-section and footer.use of linear gradient is expected as such I have used it in my project.
The following changes have been made in the given draft:
1) added anchor tags to image .
2)this is how webpage looks on my computer I am attaching link to githack.com


**HERE is a link to live preview :<a href="https://raw.githack.com/jamestucker1988/apple-microverse-project/feature/index.html">click here</a>**

**Pull request merged**